include/RAJA/exec-cuda/forall_cuda.hxx

```diff
@@ -88,9 +88,10 @@ template <typename LOOP_BODY>
 __global__ void forall_cuda_kernel(LOOP_BODY loop_body,
                                    Index_type begin,
                                    Index_type len) {
+  auto body = loop_body;
   Index_type ii = blockDim.x * blockIdx.x + threadIdx.x;
```

Comment:
I'm not sure how we are getting away with only comparing the x dimension here.  The user block may be x=19, y=27, z = 4.